finder-app/finder.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+# Counts string occurences in a directory
+# Usage:
+# ./finder.sh [dir] [string]
+# Author: James Bohn
+
+if [ $# -lt 2 ]; then
+    echo "Not enough arguments."
```

Comment:
@jbohn3353 Think about how you could make this error message more descriptive to help the user execute correctly.